<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MASC Thermistor Resistance to Tempeature Converter</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>23</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 170, 0);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string> MASC Thermistor Resistance to Temperature Converter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>170</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Resistance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Temperature</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ResistanceName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TemperatureName">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sensor Type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="SensorType">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>451</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Sontay A-10K3A1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sontay B-10k4A1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sontay C-20K6A1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sontay Q-50K6A1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sontay R-100K6A1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sontay S-SAT2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sontay T-TAT3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sontay W-SEI1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sontay Y-STA2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sontay V-SAT4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sontay Z-10K NTC</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="FTemperatureName">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>170</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
